tests/pytest/test_waves.py

```diff
@@ -57,10 +58,15 @@ def run_simulation(sim):
 
 @pytest.mark.simulator_required
 @pytest.mark.skipif(
-    os.getenv("SIM", "icarus") != "icarus",
-    reason="Skipping test because it is only for Icarus simulator",
+    sim not in ["icarus", "xcelium"],
+    reason="Skipping test because it is only for Icarus or Xcelium simulators",
 )
 def test_iverilog():
```

Comment:
Maybe something like now?
```suggestion
def test_wave_dump():
```

Reply:
Yes, good call.